daemon/kmd/wallet/driver/ledger_hid.go

```diff
@@ -207,23 +197,23 @@ func (l *LedgerUSB) USBInfo() hid.DeviceInfo {
 }
 
 // LedgerEnumerate returns all of the Ledger devices connected to this machine.
-func LedgerEnumerate() ([]LedgerUSB, error) {
+func LedgerEnumerate(log logging.Logger) ([]LedgerUSB, error) {
 	if !hid.Supported() {
 		return nil, fmt.Errorf("HID not supported")
 	}
 
 	var devs []LedgerUSB
-	for _, did := range ledgerDeviceIDs {
-		for _, info := range hid.Enumerate(did.vendor, did.product) {
-			dev, err := info.Open()
-			if err != nil {
-				return nil, err
-			}
-
-			devs = append(devs, LedgerUSB{
-				hiddev: dev,
-			})
+	for _, info := range hid.Enumerate(ledgerVendorID, 0) {
+		// Try to open the device
+		dev, err := info.Open()
+		if err != nil {
+			log.Warnf("enumerated but failed to open ledger %x", info.ProductID)
```

Comment:
nit: You might want to log the `err` as well.

Reply:
oh, good idea

Reply:
done